styles/styles.css

```
@@ -7,31 +8,479 @@
+    font-size: 12px;
+    font-weight: 400;
+}
+
+.p-question a {
+    color: var(--color-purple);
+    text-decoration: none;
+}
+
+.p-question a:hover {
+    cursor: pointer;
+    text-decoration: underline;
+}
+
+.p-project-item {
+    font-size: 14px;
+    font-weight: 300;
+}
+
+.a-view-project {
+    font-size: 9px;
+    font-weight: 500;
+    text-transform: uppercase;
+    text-decoration: none;
+    letter-spacing: 1px;
+    color: var(--color-purple)
+}
+
+.a-view-project:hover {
+    cursor: pointer;
+    text-decoration: underline;
+}
+
+.section-contact {
+    color: var(--color-light-grey);
+}
+
+label {
+    font-size: 13px;
+    font-weight: 500;
+}
+
+input, textarea {
+    font-size: 13px;
+    font-weight: 400;
+}
+
+input, textarea {
+    color: var(--color-grey);
+}
+
+
 
 /*----------------------
     3. Areas
 ----------------------*/
 
 /*------------ 3.1 Header ------------*/
+header {
+    background-image: linear-gradient(to bottom right, var(--color-blue), var(--color-purple));
+    color: var(--color-light-grey);
+
+    display: flex;
+    flex-direction: column;
+    /*padding: 20px 40px 40px 40px;*/
+    padding: 1em 2em 2em 2em;
+}
+
+nav {
+    display: flex;
+    justify-content: flex-end;
+    flex-grow: 0;
+    flex-shrink: 0;
+    padding: 10px;
+}
+
+.nav-list {
+    display: flex;
+    flex-direction: row;
+    gap: 30px;
+
+    list-style-type: none;
+}
+
+.nav-list-item {
+    color: var(--color-light-grey);
+    text-decoration: none;
+}
+
+.introduction-wrapper {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+    align-items: center;
+    flex-basis: auto;
+    gap: 20px;
+
+    /*margin: 0 150px;*/
+    /*padding: 30px 100px;*/
+    margin: 0 10em;
+    padding: 2em 6em;
+
+    text-align: center;
+}
+
+.decoration-lines {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-shrink: 0;
+    flex-grow: 0;
+    margin-top: 50px;
+    margin-bottom: -80px;
+}
+
+.line-item {
+    max-height: 100px;
+}
+
+/*------------ 3.2 Main ------------*/
+main {
+    display: flex;
+    flex-direction: column;
+    justify-content: center;
+}
+
+/*------------ 3.2.1 Section 1 - COMPANY DESCRIPTION ------------*/
+.section-company-description {
+    display: flex;
+    justify-content: center;
+    align-items: center;
+    flex-basis: auto;
+    gap: 20px;
+    margin: 0 200px;
+    padding: 60px 150px 50px 150px;
+
+    text-align: center;
+    /*border: 1px solid red;*/
+}
+
+.p-question {
+    padding-top: 70px;
+}
+
+/*------------ 3.2.2 Section 2 - PROJECTS ------------*/
+.section-projects {
+    background-color: var(--color-light-grey);
+}
+
+.outer-container {
+    display: flex;
+    flex-direction: column;
+    margin: 50px;
+}
+
+.project-item-1, .project-item-3 {
+    display: flex;
+    flex-direction: row;
+    justify-content: flex-start;
+    align-items: center;
+    gap: 50px;
+    /*border: 1px solid red;*/
+    padding: 0 0 80px 0;
```

Als je geen padding-top, -right en -left hebt om te overschrijven, dan prefereer ik padding-bottom:
```suggestion
    padding-bottom: 80px;
```